### PR TITLE
feat(lobste.rs): init

### PR DESCRIPTION
**Pull request by steinuil**
## 🎉 Theme for [Lobsters](https://lobste.rs) 🎉

Lobsters is a computing-focused community centered around link aggregation and discussion.

This is how the theme looks on my computer.

Screenshot of Lobsters on Firefox

## 💬 Additional Comments 💬

The SVG icon comes [from the lobsters repo](https://github.com/lobsters/lobsters/blob/master/app/assets/images/logo-color.svg).

Not all the CSS variables defined in the CSS file are mapped, I just changed the ones I saw were used in all the pages I tested (which is most of them).

Also, I had to use `fade()` to get the colors for the tags.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - [x] `catppuccin.user.css` - all the CSS for the userstyle, based on the
        template.
  - [x] `preview.webp` - composite image of all four individual flavor screenshots (taken with the default accent color of mauve) stitched together, generated via [Catwalk](https://github.com/catppuccin/catwalk).

### CODE REVIEW

File: styles/lobste.rs/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+          '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><g><path fill="@{accent-color}" d="M16,16c-5.33,0-10.67,0-16,0C0,10.67,0,5.33,0,0c5.33,0,10.67,0,16,0C16,5.33,16,10.67,16,16z"/><path fill="@{base}" d="M3.5,13.45c0-0.1,0-0.2,0-0.3c0-0.11,0-0.22,0-0.33c0-0.11,0-0.1,0.1-0.11c0.23-0.02,0.46-0.03,0.69-0.07   c0.46-0.08,0.67-0.31,0.74-0.78c0.02-0.14,0.03-0.29,0.03-0.43c0-2.25,0-4.5,0-6.74c0-0.18-0.02-0.36-0.04-0.53   C4.99,3.85,4.79,3.66,4.5,3.55c-0.27-0.1-0.55-0.11-0.83-0.13c-0.05,0-0.1,0-0.15,0c0-0.03,0-0.06,0-0.08c0-0.19,0-0.38,0-0.57   c0-0.07,0.02-0.09,0.09-0.09c1.04,0,2.07,0,3.11,0c0.71,0,1.42,0,2.13,0c0.07,0,0.1,0.01,0.1,0.09c-0.01,0.19,0,0.37,0,0.56   c0,0.03,0,0.05,0,0.08c-0.14,0.01-0.28,0-0.41,0.02c-0.2,0.03-0.4,0.06-0.59,0.11C7.6,3.65,7.42,3.89,7.38,4.22   c-0.02,0.15-0.03,0.3-0.03,0.45c0,2.13,0,4.26,0,6.39c0,0.27,0.03,0.54,0.06,0.8c0.03,0.29,0.22,0.48,0.49,0.58   c0.23,0.09,0.48,0.11,0.72,0.12c0.63,0.02,1.25,0.01,1.86-0.15c0.88-0.22,1.47-0.77,1.76-1.64c0.09-0.28,0.13-0.57,0.2-0.86   c0.01-0.02,0.03-0.06,0.05-0.06c0.23,0,0.42,0,0.66,0c-0.02,1.2,0.02,2.4,0,3.6C9.94,13.45,6.71,13.45,3.5,13.45z"/></g></svg>'
+        )
+      );
```
Comment (uncenter) [inline styles/lobste.rs/catppuccin.user.css:135]: You shouldn't need to wrap this in the `e` function.

File: styles/lobste.rs/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    --color-fg-shape: @surface2;
+
+    --color-tag-bg: fade(@yellow, 20%);
+    --color-tag-border: fade(@yellow, 20%);
+
+    --color-lobsters-tag-special-bg: fade(@red, 30%);
+    --color-lobsters-tag-special-border: fade(@red, 30%);
+
+    --color-tag-media-bg: fade(@blue, 20%);
+    --color-tag-media-border: fade(@blue, 30%);
```
Comment (uncenter) [inline styles/lobste.rs/catppuccin.user.css:95]: Just curious what these variables theme specifically and how it looks?

**Review by uncenter — CHANGES_REQUESTED** [review]
From the preview image it looks like tags on posts aren't themed.

File: styles/lobste.rs/catppuccin.user.css (hunk 1 above)
Comment (steinuil) [reply styles/lobste.rs/catppuccin.user.css:135]: Huh, when I tried it earlier, if I didn't wrap it with `e` LESS would throw an error along the lines of "invalid % without a number", but it's not doing that anymore. I'll remove it.

File: styles/lobste.rs/catppuccin.user.css (hunk 2 above)
Comment (steinuil) [reply styles/lobste.rs/catppuccin.user.css:95]: Those are the post tags! This is how they look styled vs unstyled:

Screenshot 2024-11-18 at 11-21-32 Tags Lobsters

Screenshot 2024-11-18 at 11-21-55 Tags Lobsters

File: styles/lobste.rs/catppuccin.user.css (hunk 1 above)
Comment (steinuil) [reply styles/lobste.rs/catppuccin.user.css]: Right, when I was testing I hadn't replaced the colors yet. Anyway, I removed it now.

**Comment by uncenter** [comment]
Would it be possible to get an invitation? I'd like to be able to check that every page is themed, even the password/login protected ones.

**Comment by steinuil** [comment]
Sure, I just need an email address. You can send me an email at steen (at) neets.work.

**Review by uncenter — CHANGES_REQUESTED** [review]
CleanShot 2024-11-19 at 17 45 57

This button is unthemed - that's it though! Once that is fixed we can merge, really nice work on this - and thanks for the invite :D

**Comment by steinuil** [comment]
All done! Thanks :)

Do the commits get squashed when the branch is merged or should I do that myself?

**Comment by uncenter** [comment]
We squash merge PRs, nothing needed from you there.

**Comment by uncenter** [comment]
We squash merge PRs, nothing needed from you there.

File: styles/lobste.rs/catppuccin.user.css [hunk 3]
```diff
@@ -0,0 +1,161 @@
+/* ==UserStyle==
+@name Lobsters Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/lobste.rs
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/lobste.rs
+@version 1.0.0
```
Comment (uncenter) [inline styles/lobste.rs/catppuccin.user.css:5]:
```suggestion
@version 0.1.0
```

**Review by uncenter — DISMISSED** [review]

**Comment by isabelroses** [comment]
fixed merge conflict :)

**Review by isabelroses — APPROVED** [review]

**Comment by isabelroses** [comment]
Thanks for your hard work on the new userstyle @steinuil ❤️ 

Soon after this is merged you will be invited to the userstyles maintainers team. That will grant you the ability to merge PRs for your userstyle.

Welcome to the team!

**Pull request merged**